presto-product-tests/src/main/java/com/facebook/presto/tests/SystemConnectorTests.java

```diff
@@ -29,15 +32,23 @@
     @Test(groups = SYSTEM_CONNECTOR)
     public void selectRuntimeNodes()
     {
-        assertThat(query("SELECT node_id, http_uri, node_version, state FROM system.runtime.nodes"))
-                .hasColumns(LONGNVARCHAR, LONGNVARCHAR, LONGNVARCHAR, LONGNVARCHAR)
-                .hasAnyRows();
+        String sql = "SELECT node_id, http_uri, node_version, state FROM system.runtime.nodes";
+        if (usingOpenSourceJdbcDriver()) {
+            assertThat(query(sql))
+                    .hasColumns(LONGNVARCHAR, LONGNVARCHAR, LONGNVARCHAR, LONGNVARCHAR)
+                    .hasAnyRows();
+        }
+        else {
+            assertThat(query(sql))
+                    .hasColumns(VARCHAR, VARCHAR, VARCHAR, VARCHAR)
```

Comment:
I checked with Simba, online JDBC docs, and Presto CLI.  The actual column type in Presto is VARCHAR, so that is the correct value.  I filed [GitHub 4797](https://github.com/facebook/presto/issues/4797) because the open source driver is returning LONGNVARCHAR when it should be returning VARCHAR.

I could eliminate the if and let it fail with the open source driver, but I thought that might be annoying since we use the open source driver for the bulk of our tests.